src/audio/CMakeLists.txt

```diff
@@ -70,6 +70,19 @@ if(NOT CONFIG_LIBRARY)
 		target_link_libraries(sof PRIVATE c)
 		target_link_libraries(sof PRIVATE m)
 	endif()
+
+	if(CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING)
+		add_local_sources(sof
+		        google_rtc_audio_processing.c
+		)
+		target_link_libraries(sof PRIVATE google_rtc_audio_processing)
+		target_link_libraries(sof PRIVATE google_rtc_audio_processing_tuning)
+		target_link_libraries(sof PRIVATE c++)
+		target_link_libraries(sof PRIVATE c++abi)
+		target_link_libraries(sof PRIVATE m)
+		target_link_libraries(sof PRIVATE c)
```

Comment:
Should we make the std link libraries a global kconfig option as they may be used by other modules e.g. I have some POC code that builds the Tensorflow lite microspeech example that uses the same libraries and would be able to reuse ?

Reply:
Yea that sounds like a good idea to make it a general select in the kconfig to avoid linking multiple times over

Reply:
I think we are OK atm, lets cross that when we get a better toolchain.